js/models/ShippingAddress.js

```diff
@@ -0,0 +1,16 @@
+import BaseModel from './BaseModel';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      Name: '',
+      Company: '',
+      AddressLineOne: '',
+      AddressLineTwo: '',
+      City: '',
+      State: '',
+      Country: '',
+      PostalCode: '',
```

Comment:
I think Washington has some research on it Chris was going to implement. There's a shared doc with it someplace.